pkg/controllers/jobset_controller_test.go

```diff
@@ -899,14 +1068,15 @@ func TestCalculateReplicatedJobStatuses(t *testing.T) {
 }
 
 type makeJobArgs struct {
-	jobSetName        string
-	replicatedJobName string
-	jobName           string
-	ns                string
-	replicas          int
-	jobIdx            int
-	restarts          int
-	topology          string
+	jobSetName           string
```

Comment:
We can do this as a followup, but we are using the wrapper pattern, we shouldn't need a makeJobArgs struct, we should be able to do something like

```
makeJob(name, namespace)
  .ReplicatedJobName(replicatedJobName)
  .Replicas(n)
```